src/Angor/Client/Storage/ClientStorage.cs

```diff
@@ -112,83 +101,66 @@ public List<FounderProject> GetFounderProjects()
 
     public void UpdateFounderProject(FounderProject project)
     {
-        var projects = _storage.GetItem<List<FounderProject>>("founder-projects");
-        
-        var item = projects.FirstOrDefault(f => f.ProjectInfo.ProjectIdentifier == project.ProjectInfo.ProjectIdentifier);
+        var projects = _storage.GetItem<List<FounderProject>>("founder-projects") ?? new List<FounderProject>();
+
+        var existingIndex = projects.FindIndex(p =>
+            p.ProjectInfo.ProjectIdentifier == project.ProjectInfo.ProjectIdentifier &&
```

Comment:
no this is wrong, ProjectIdentifier must be unique if we have more then one it means we are incorrectly using the index else where